pages/sdk/relay-kit/reference/migrating-to-v3.md

```diff
@@ -0,0 +1,35 @@
+# Migrating to v3
+
+This guide references the major changes between v2 and v3 to help those migrating an existing app.
+
+## Removing adapters
+
+We have removed the concept of adapters from the `protocol-kit` to simplify the library. Instead of using specific library adapters, we will now use an internal `SafeProvider` object to interact with the Safe. This `SafeProvider` will be created using:
```

Comment:
```suggestion
We have removed the concept of adapters from the `protocol-kit` to simplify the library. Instead of using specific library adapters, we use now an internal `SafeProvider` object to interact with the Safe. This `SafeProvider` will be created using:
```